woocommerce/woocommerce
# Lower opacity for payments section to work across light and dark backgrounds

### Changes proposed in this Pull Request:

This change uses the secondary color created using alpha opacity to allow the background color of the page to come through so that text is readable on both light and dark background colors in the payments section of the classic checkout.


#### Before
<img width="1025" alt="Screenshot 2024-10-22 at 3 06 16 PM" src="https://github.com/user-attachments/assets/5b66b374-9d25-483c-b493-a68405080afa">
<img width="1023" alt="Screenshot 2024-10-22 at 3 06 01 PM" src="https://github.com/user-attachments/assets/1ee94892-fb28-4a61-af17-ff5b7134cc26">

#### After

<img width="1024" alt="Screenshot 2024-10-22 at 2 44 56 PM" src="https://github.com/user-attachments/assets/f5ce42c1-eeaf-4b25-adcd-6d881b4e2794">
<img width="1023" alt="Screenshot 2024-10-22 at 2 45 34 PM" src="https://github.com/user-attachments/assets/29719da4-9bf0-4851-a94e-e762555af641">


Closes #52236  .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Activate a theme with style presets, like Twenty Twenty-Four
2. Navigate to Appearance->Editor->Styles.
3. Select a dark background with light text color style.
4. Create a page with a classic checkout block
5. Add an item to the cart
6. Navigate to your classic checkout page
7. Note the text and background color of the payment section is readable
8. Navigate to Appearance->Editor->Styles
9. Select a light background color with dark text color
10. Check that the payments section is still readable



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
